docs/nix-flakes.adoc

```diff
@@ -126,23 +126,22 @@ $ nix flake new ~/.config/nixpkgs -t github:nix-community/home-manager
 2. Install Home Manager and apply the configuration by
 +
 [source,console]
-----
-$ nix build --no-link <flake-uri>#homeConfigurations.jdoe.activationPackage
-$ "$(nix path-info <flake-uri>#homeConfigurations.jdoe.activationPackage)"/activate
-----
+$ nix run <flake-uri>#homeConfigurations.jdoe.activationPackage
 +
 Substitute `<flake-uri>` with the flake URI of the configuration flake.
 If `flake.nix` resides in `~/.config/nixpkgs`,
 `<flake-uri>` may be `~/.config/nixpkgs`
 as a Git tree or `path:~/.config/nixpkgs` if not.
 
-3. Since the release `21.05`,
-building a flake-based configuration is as simple as
+3. Once `home-manager` is installed, building a flake-based configuration is as simple as
 +
 [source,console]
-$ home-manager switch --flake '<flake-uri>#jdoe'
+$ home-manager switch
 +
-once home-manager is installed.
+If your `flake.nix` does not reside in `~/.config/nixpkgs`, you must instead use
++
+[source,console]
+$ home-manager switch --flake '<flake-uri>#jdoe'
```

Comment:
See https://github.com/nix-community/home-manager/issues/3640#issuecomment-1416851735: the default path is not great, probably better to keep `--flake` explicitly

Reply:
I wasn't aware, thanks!
From the look of things, there are no concrete plans on electing a better default, right?